tests/test_buildhistory.py

```diff
@@ -8,22 +8,23 @@ class TestBuildHistory:
 
     def test_buildhistoryurlvisible(self):
```

Comment:
Add _ to function name  for better readability

Reply:
Lets skip it this time

Reply:
Done